### ISSUE TITLE
List of "first PR" issues 

### ISSUE THREAD

**Issue by HaleySchuhl** [danforthcenter/plantcv#999]
**Start a discussion about anything you would like**
In regards to the team-coding-day that we have planned for March 22nd, 2023; these are all issues that would require very few lines of code get changed to resolve. Please edit this comment... add more issues, assign yourself when you start a task, and mark ask completed once PR is merged into the correct branch. 

- [x] https://plantcv.readthedocs.io/en/4.x/visualize_ovelay_two_imgs/ Update code block example to reflect the function being within the `visualize` sub-package.  @jmgordon1223 
- [x] https://plantcv.readthedocs.io/en/latest/visualize_colorspace/ Typo in docs --> `originial_img` @JoeDuenwald 
- [x]  https://plantcv.readthedocs.io/en/latest/installation/ No longer using 3.7 but we are testing 3.10 so update docs @joshqsumner 
- [x] https://plantcv.readthedocs.io/en/stable/analyze_thermal_values/ Typo in docs --> `thermal_hist  = pcv.analyze_thermal_values(thermal_array=thermal_img, maks=mask, histplot=True, label="default")` @lchavez037 
- [x] https://plantcv.readthedocs.io/en/stable/texture_threshold/ Hyperlink to scikit image documentation rather than including link as plain text when describing `texture_method` context. @kmurphy61 
- [x] #1063
- [ ]

**Comment by nfahlgren** [comment]
Improve installation docs? Maybe include Jupyterlab desktop client?

**Comment by HaleySchuhl** [comment]
Deprecation warnings:
```plantcv/plantcv/median_blur.py:7
  /home/runner/work/plantcv/plantcv/plantcv/plantcv/median_blur.py:7: DeprecationWarning: Please use `median_filter` from the `scipy.ndimage` namespace, the `scipy.ndimage.filters` namespace is deprecated.

plantcv/plantcv/fill_holes.py:8
  /home/runner/work/plantcv/plantcv/plantcv/plantcv/fill_holes.py:8: DeprecationWarning: Please use `binary_fill_holes` from the `scipy.ndimage` namespace, the `scipy.ndimage.morphology` namespace is deprecated.
```


- [ ] `scipy.ndimage.morphology`
- [ ] `scipy.ndimage.filters`

**Comment by nfahlgren** [comment]
We accidentally dropped testing coverage by 1 line in `main`. Could be an easy fix in `analyze_objects`

**Comment by HaleySchuhl** [comment]
Update https://plantcv.readthedocs.io/en/latest/pr_review_process/#step-4-review-the-tests  

The actual command is :

**Comment by jmgordon1223** [comment]
@jmgordon1223 analyze thermal values change print to plot

**Comment by lacostag** [comment]
- [ ] @lacostag (apply mask)

**Comment by JoeDuenwald** [comment]
- [ ] @JoeDuenwald (analyze_stem)

**Comment by lchavez037** [comment]
@lchavez037 (auto_crop)

**Comment by JoeDuenwald** [comment]
- [ ] @JoeDuenwald (analyze_thermal_values)

**Comment by sdkenney42** [comment]
Sam- change print to plot in y-axis-psuedolandmarks

**Comment by lacostag** [comment]
- [ ] @lacostag auto_crop

**Comment by lchavez037** [comment]
@lchavez037 acute_vertex.md

**Comment by JoeDuenwald** [comment]
@JoeDuenwald watershed.md

**Comment by leowlima** [comment]
- [ ] @leowlima gaussian_threshold.md

**Comment by leowlima** [comment]
- [ ] @leowlima HistEqualization

**Comment by sdkenney42** [comment]
- [ ] @sdkenney42 watershed

**Comment by JoeDuenwald** [comment]
- [ ] @JoeDuenwald erode

**Comment by jmgordon1223** [comment]
- [ ] @jmgordon1223  Canny Edge Detection print to plot

**Comment by lacostag** [comment]
- [ ] cluster_contours

**Comment by lchavez037** [comment]
@lchavez037 skeletonize print to plot

**Comment by leowlima** [comment]
- [ ] @leowlima image_add.md

**Comment by sdkenney42** [comment]
- [ ] @sdkenney42 triangle_threshold

**Comment by leowlima** [comment]
- [ ] @leowlima image_fusion

**Comment by lacostag** [comment]
- [ ] @lacostag check_cycles

**Comment by jmgordon1223** [comment]
- [ ] @jmgordon1223 (closing)

**Comment by leowlima** [comment]
- [ ] @leowlima image_subtract

**Comment by leowlima** [comment]
- [ ] @leowlima landmark_reference_pt_dist

**Comment by lchavez037** [comment]
@lchavez037 spatial_clustering

**Comment by JoeDuenwald** [comment]
- [ ] @JoeDuenwald custom_range_threshold

**Comment by leowlima** [comment]
- [ ] @leowlima laplace_filter

**Comment by sdkenney42** [comment]
- [ ] @sdkenney42 spectral_index

**Comment by jmgordon1223** [comment]
- [ ] @jmgordon1223  change print to plot in (Cluster Contours and Split Images)

**Comment by JoeDuenwald** [comment]
- [ ] @JoeDuenwald crop

**Comment by lchavez037** [comment]
@lchavez037 stdev_filter.md

**Comment by sdkenney42** [comment]
- [ ] @sdkenney42 invert

**Comment by jmgordon1223** [comment]
- [ ] @jmgordon1223 print to plot (rgb to gray)

**Comment by leowlima** [comment]
Mean_Threshold.md

**Comment by sdkenney42** [comment]
- [ ] @sdkenney42 image_add

**Comment by leowlima** [comment]
median_blur

**Comment by sdkenney42** [comment]
- [ ] @sdkenney42 get_nir

**Comment by sdkenney42** [comment]
- [ ] @sdkenney42 find_objects

**Comment by leowlima** [comment]
naive_bayes_classifier
